src/screens/Inventory/components/InventoryUpdateCard.tsx

```diff
@@ -0,0 +1,53 @@
+import { Card, Typography } from '@material-ui/core';
+import { createStyles, Theme, withStyles } from '@material-ui/core/styles';
+import React from 'react';
+import { useSelector } from 'react-redux';
+import { selectProductByInventoryId } from '../../../lib/redux/inventoryData';
+import { EMPTY_PRODUCT } from '../../../lib/redux/inventoryDataSlice';
+import { RootState } from '../../../lib/redux/store';
+
+const styles = (theme: Theme) =>
+  createStyles({
+    arrow: {
+      color: theme.palette.text.primary,
+    },
+    cardContent: {
+      flex: 1,
+      padding: 16,
+    },
+    cardContainer: {
+      border: `1px solid ${theme.palette.divider}`,
+      boxShadow: 'none',
+      borderRadius: 10,
+      display: 'flex',
+      marginBottom: 10,
+    },
+    arrowSpacing: {
+      padding: 0,
+    },
+  });
+
+interface InventoryUpdateCardProps {
+  classes: { arrow: string; cardContent: string; cardContainer: string; arrowSpacing: string };
+  inventoryId: string;
+  updatedQuantity: number;
+  createdAt: string;
+  userId: string;
+}
+
+function InventoryUpdateCard(props: InventoryUpdateCardProps) {
+  const { classes, inventoryId, updatedQuantity, createdAt, userId } = props;
+  const product = useSelector((state: RootState) => selectProductByInventoryId(state, inventoryId)) || EMPTY_PRODUCT;
+
+  return (
+    <Card className={classes.cardContainer}>
+      <div className={classes.cardContent}>
+        <Typography variant="body1">{`${product.name}, ${updatedQuantity} ${product.unit}(s)`}</Typography>
+        <Typography variant="body1">{`Created at: ${createdAt}`}</Typography>
```

Comment:
I imagine we want to eventually change this to the timezone of the user. The only one I see that isn't in the appropriate timezone is the one in `createdAt`. There are other dates I think that are unchanged too, so we could change this in a future PR, but since it's quick I'd rather not have to do more work down the line

Screenshot: I created this at 8AM PST 4/3/2021
![image](https://user-images.githubusercontent.com/35501399/113482383-686b7000-9453-11eb-9890-05d40b445464.png)

Reply:
I ended up reformatting the way we do dates and times entirely by updating the helper function in `momentUtils` to return a string format that is sensitive to both language and timezone. I was worried about different locales having different formats/orderings of month/day/year etc so I think this resolves that issue.

**Myanmar**
![image](https://user-images.githubusercontent.com/21160510/113486858-a07dad80-9469-11eb-80ca-a8ae9b5992c3.png)

**San Francisco**
![image](https://user-images.githubusercontent.com/21160510/113486862-a70c2500-9469-11eb-89be-7ca1dc76a29d.png)

**Tokyo (just for contrast)**
![image](https://user-images.githubusercontent.com/21160510/113486870-b1c6ba00-9469-11eb-992d-aedb57c3d373.png)